lib/usd/translators/shading/mtlxBaseWriter.cpp

```diff
@@ -75,23 +74,6 @@ REGISTER_SHADING_MODE_EXPORT_MATERIAL_CONVERSION(
     TrMtlxTokens->niceName,
     TrMtlxTokens->exportDescription);
 
-// Register symmetric writers:
-TF_REGISTRY_FUNCTION(UsdMayaShaderWriterRegistry)
-{
-    UsdMayaSymmetricShaderWriter::RegisterWriter(
```

Comment:
Moved to mtlxSymetricShaderWriter.cpp since there are NodeGraph considerations to support.